components/core/src/server/installable-endpoints/ingest.ts

```diff
@@ -0,0 +1,20 @@
+import { CustomRequestHandler } from "../customrequesthandler";
+import { IntegrationApi, OfferChange } from "../../coreapi";
+import { OprServer } from "../../../build/server/oprserver";
+
+
+export const ingestEndpoint = (api: IntegrationApi, s: OprServer): any => { 
```

Comment:
Please change the return type of this function to "unknown" (or, even better, the upcoming opr-core/JsonValue interface). Our lint rules disallow "any" types except when absolutely necessary.

Thanks!